pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -267,7 +267,8 @@ stdenv.mkDerivation ((drvAttrs config hostPlatform.platform kernelPatches config
   hardeningDisable = [ "bindnow" "format" "fortify" "stackprotector" "pic" ];
 
   makeFlags = commonMakeFlags ++ [
-    "HOSTCC=${buildPackages.stdenv.cc.targetPrefix}gcc"
+    "CC=${stdenv.cc}/bin/cc"
```

Comment:
still need the `stdenv.cc.targetPrefix` like below, but otherwise good. Thanks!

Reply:
OK, I assumed `stdenv.cc` is always prefix-less.